requirements.txt

```diff
@@ -1,3 +1,7 @@
 uiautomator2
 uiautomator
 colorama
+
```

Comment:
This is not supposed to be part of the project's requirement. Please remove it.

Reply:
I've imported uiatomator2 in utils.py, in order to use these functions. (connect/info/press).
Why you say it's not a part of the project? We switch from uiautomator to uiatomator 2 and introduces --old tag for those that had problems with it 👍

Reply:
I meant that this files "requirements.txt" should not be changed since there is no new dependency being added.